### PR TITLE
refactor: add unit test in notification component

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: src/components/Notification/__test__/icon.spec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <Icon icon={<svg />} />,
+        );
+        expect(component.find('svg').exists()).toBe(true);
+    });
+    it('should render the "InfoIcon" when the string "info" is passed as icon', () => {
+        const component = mount(
+            <Icon icon="info" />,
+        );
+        expect(component.find('InfoIcon').exists()).toBe(true);
+    }); it('should render the "ErrorIcon" when the string "error" is passed as icon', () => {
```
Comment (LeandroTorresSicilia) [inline src/components/Notification/__test__/icon.spec.js]: what is this?
new line

File: src/components/Notification/__test__/icon.spec.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        const component = mount(
+            <Icon icon="error" />,
+        );
+        expect(component.find('ErrorIcon').exists()).toBe(true);
+    }); it('should render the "WarningIcon" when the string "warning" is passed as icon', () => {
```
Comment (LeandroTorresSicilia) [inline src/components/Notification/__test__/icon.spec.js]: same^^

File: src/components/Notification/__test__/icon.spec.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        const component = mount(
+            <Icon icon="warning" />,
+        );
+        expect(component.find('WarningIcon').exists()).toBe(true);
+    }); it('should render the "SuccessIcon" when the string "success" is passed as icon', () => {
```
Comment (LeandroTorresSicilia) [inline src/components/Notification/__test__/icon.spec.js]: same^^

File: src/components/Notification/__test__/notification.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Notification from './../index';
+
+describe('<Notification/>', () => {
+    it('should call onClick function when someone click over', () => {
```
Comment (LeandroTorresSicilia) [inline src/components/Notification/__test__/notification.spec.js]: should fire an event when the close button is clicked

File: src/components/Notification/icons/index.js [hunk 5]
```diff
@@ -23,7 +30,7 @@ export default function Icon({ icon }) {
     if (typeof icon === 'string') {
         return (
             <div className={getClassName()}>
```
Comment (LeandroTorresSicilia) [inline src/components/Notification/icons/index.js]: sorry, it is better if we return this container inside VariantIcon, thus if a wrong string is passed it is not rendered. So VariantIcon would be:
```
function VariantIcon({ icon }) {
    if (iconMap[icon]) {
        return (
            <div className={getClassName()}>
                {iconMap[icon]()}
            </div>
        );
    }
    return null;
}
```
Also you need to test this case (when a wrong string is passed), that null is returned or nothing is rendered.
